live-build/misc/ansible-roles/appliance-build.dcenter/tasks/main.yml

```diff
@@ -87,4 +88,10 @@
     regexp: '^preserve_hostname: false'
     line: 'preserve_hostname: true'
 
+# Update number of NFS threads to 64
+- lineinfile:
+    path: /etc/default/nfs-kernel-server
+    regexp: '^RPCNFSDCOUNT=8'
```

Comment:
I'm OK with this, but just want to point out that this will not work if the default of `8` is ever changed. Perhaps we should change `regexp` to `^RPCNFSDCOUNT=`? This way, it's updated no matter what the default value is.